app/domain/logs.rb

```diff
@@ -64,6 +64,36 @@ module Authentication
       code: "CONJ00030D"
     )
 
+    ExtractingRestrictionsFromResource = ::Util::TrackableLogMessageClass.new(
+      msg: "Extracting resource restrictions for authenticator '{0-authn-name}' from host '{1-host-name}'",
+      code: "CONJ00039D"
+    )
+
+    ExtractedResourceRestrictions = ::Util::TrackableLogMessageClass.new(
+      msg: "Resource restrictions were extracted: '{0-restrictions-names}'",
+      code: "CONJ00040D"
+    )
+
+    ValidatingResourceRestrictionsConfiguration = ::Util::TrackableLogMessageClass.new(
+      msg: "Validating resource restrictions configuration",
+      code: "CONJ00041D"
+    )
+
+    ValidatedResourceRestrictionsConfiguration = ::Util::TrackableLogMessageClass.new(
+      msg: "Resource restrictions configuration validated",
+      code: "CONJ00042D"
+    )
+
+    ValidatingResourceRestrictionsValues = ::Util::TrackableLogMessageClass.new(
+      msg: "Validating resource restriction values",
+      code: "CONJ00044D"
+    )
+
+    ValidatedResourceRestrictionsValues = ::Util::TrackableLogMessageClass.new(
+      msg: "Resource restriction values validated",
+      code: "CONJ00045D"
+    )
+
```

Comment:
it seems that you have a `ResourceRestrictions` module for other classes. can these classes be in that module as well?

Reply:
same goes for errors

Reply:
done